spec/timezone.html

```diff
@@ -5,6 +5,99 @@
   <h1>Temporal.TimeZone Objects</h1>
   <p>A Temporal.TimeZone object is an immutable Object referencing a time zone.</p>
 
+  <emu-clause id="sec-time-zone-names">
+    <h1>Time Zone Names</h1>
+
+    <emu-note type="editor">
+      <p>
+        The normative intention of this section is that an implementation must support UTC and a local time zone, which may also be UTC, or a named time zone like `"Europe/London"`, or an offset time zone like `"+01:00"`; and that the local time zone must in any case be consistent with the time zone offsets that can be determined using `Date.prototype.getTimezoneOffset()`, `Date.prototype.toString()`, and `Date.prototype.toTimeString()`.
+        The exact formal definition of "local time zone" is still to be determined.
+      </p>
+    </emu-note>
+
+    <p>
+      An ECMAScript implementation must support a number of built-in time zones.
+      At a minimum, implementations must support a built-in time zone named *"UTC"*.
+      If the return value of DefaultTimeZone is different from *"UTC"*, and does not satisfy the syntax of a |TimeZoneNumericUTCOffset|, then implementations must support that time zone as a built-in time zone as well.
+      In addition, implementations may support any number of other built-in time zones.
+    </p>
+    <p>
+      Built-in time zones may be <dfn>named</dfn> time zones, represented by Strings for which IsValidTimeZoneName returns *true*.
```

Comment:
```suggestion
      Built-in time zones may be <dfn>named time zones</dfn>, represented by Strings for which IsValidTimeZoneName returns *true*.
```

otherwise you get links in text like "The abstract operation GetOption extracts the value of the property **named** _property_…"